src/Microsoft.AspNet.Mvc.Razor/DependencyInjection/MvcRazorMvcCoreBuilderExtensions.cs

```diff
@@ -119,7 +120,9 @@ public static IMvcCoreBuilder AddRazorViewEngine(this IMvcCoreBuilder builder)
         // Internal for testing.
         internal static void AddRazorViewEngineServices(IServiceCollection services)
         {
-            if (CompilationServices.Default != null)
+            var isDotnet = CompilationServices.Default == null;
```

Comment:
change the variable name to be less about dotnet and more about compilation services missing.